docs/Authentication.md

```diff
@@ -82,6 +82,59 @@ Now the admin is secured: The user can be authenticated and use their credential
 
 If you have a custom REST client, don't forget to add credentials yourself.
 
+## Handling External Authentication Services Callbacks
+
+When using external authentication services such as those implementing OAuth, you usually need a callback route. React-admin provides a default one at `/login-callback`. It will call the `AuthProvider.handleLoginCallback` method that may validate the params received from the URL and redirect users to any page (the home page by default) afterwards.
```

Comment:
please add details about the expected return type (for redirections)

Reply:
Done in https://github.com/marmelab/react-admin/pull/8457/files#diff-b3d8b3b7ae1bfb881040b85c6ce05bad019fe65b83538fb0c80f3aef787ac8e9